api/Clipboard.json

```diff
@@ -187,34 +178,13 @@
             "edge": {
               "version_added": "79"
             },
-            "firefox": [
-              {
-                "version_added": "87",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "dom.events.asyncClipboard.clipboardItem",
-                    "value_to_set": "true"
-                  }
-                ],
-                "notes": "Writing to the clipboard is available without permission in secure contexts and browser extensions, but only from user-initiated event callbacks. Browser extensions with the <code>\"clipboardWrite\"</code> permission can write to the clipboard at any time."
-              },
-              {
-                "version_added": "63",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "dom.events.asyncClipboard.dataTransfer",
-                    "value_to_set": "true"
-                  }
-                ],
-                "partial_implementation": true,
-                "notes": [
-                  "This method accepts a <code>DataTransfer</code> object instead of an array of <code>ClipboardItem</code> objects.",
-                  "Writing to the clipboard is available without permission in secure contexts and browser extensions, but only from user-initiated event callbacks. Browser extensions with the <code>\"clipboardWrite\"</code> permission can write to the clipboard at any time."
-                ]
-              }
-            ],
+            "firefox": {
+              "version_added": "preview",
+              "notes": [
+                "Writing to the clipboard is available without permission in secure contexts and browser extensions, but only when the contexts have <a href=\"https://developer.mozilla.org/docs/Web/Security/User_activation#transient_activation\">transient activation</a>. Browser extensions with the <code>\"clipboardWrite\"</code> permission can write to the clipboard at any time.",
+              "From version 122, Firefox supports MIME types: <code>text/plain<code>, <code>text/html<code> and <code>image/png<code>."
```

Comment:
@Elchi3 This comes from a note in the docs [](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/write)

> **Note:** For parity with Google Chrome, Firefox only allows this function to work with text, HTML, and PNG data.

I was wondering if we might expect this list to grow? In that case a subfeature for each of the supported types better than these notes? The argument for is precision and easy updates in future. OK with you to add subfeature instead for the three types?

Reply:
None of the `<code>` tags are closed. Node.js complained about that while testing.

Reply:
Thanks.
```suggestion
              "From version 122, Firefox supports MIME types: <code>text/plain</code>, <code>text/html</code> and <code>image/png</code>."
```